src/communities/communities.controllers.js

```diff
@@ -1,19 +1,59 @@
 import { status } from '../../config/response.status.js';
 import { response } from '../../config/response.js';
-import { getCommunitiesService } from './communities.service.js';
-
-export const getCommunitiesController = async (req, res, next) => {
-    const { page = 1, size = 10 } = req.query;  // req.body 대신 req.query 사용
+import { joinCommunityService } from './communities.service.js';
+import { JoinCommunityDTO } from './communities.dto.js';
 
+// 커뮤니티 가입 컨트롤러
+export const joinCommunityController = async (req, res, next) => {
     try {
-        const communitiesData = await getCommunitiesService(parseInt(page), parseInt(size));
+        const { community_id } = req.body; // community_id는 요청 본문에서 받음
+        const user_id = req.user.id; // 미들웨어에서 설정된 사용자 ID 사용
+
+        if (!community_id) {
+            return res.status(status.BAD_REQUEST.status).send(response(
+                status.BAD_REQUEST,
+                null,
+                `필요한 정보가 누락되었습니다: ${status.BAD_REQUEST.message}`
+            ));
```

Comment:
```suggestion
            return res.send(response(status.BAD_REQUEST));
```

이렇게 바로 상태코드 반환해주시면 돼요!

Reply:
전부 바꿨습니다!